index.bs

```diff
@@ -1457,6 +1459,81 @@ Resignation from a Group</h3>
 	See “Exclusion and Resignation from the Working Group” in the W3C Patent Policy [[PATENT-POLICY]]
 	for information about obligations remaining after resignation from certain groups.
 
+<h3 id="tooling">
+Tooling</h3>
+
+	For [=W3C Groups=] operating under this Process,
+	a core operating principle is to allow access across disabilities,
+	across country borders,
+	and across time.
+	Thus in order to allow all would-be participants to effectively participate,
+	to allow future participants and observers to understand the rationale and origins of current decisions,
+	and to guarantee long-lived access to its publications,
+	W3C requires that:
+
+	<ul>
+		<li>
+			All reports, publications, or other deliverables
+			produced by the group for public consumption
+			(i.e. intended for use or reference outside its own membership)
+			<em class=rfc2119>should</em> be published and promoted at a W3C-controlled URL,
+			and backed up by W3C systems
+			such that if the underlying service is discontinued,
+			W3C can continue to serve such content without breaking incoming links
+			or other key functionality.
+
+		<li>
+			All reports, publications, or other deliverables
+			produced by the group for public consumption
+			<em class=rfc2119>should</em> follow best practices for accessibility worldwide
+			and for persons with disabilities.
+			Network access to w3.org itself may be assumed.
+
+		<li>
+			Persistent text-based discussions sponsored by the group
+			pertaining to their work
+			and intended to be referenceable by all group members <em class=rfc2119>should</em>--
+			and official meeting minutes and especially records of decisions made <em class=rfc2119>must</em>--
+			be archived by W3C for future reference.
```

Comment:
(I'd probably avoid the parenthetical – – by saying it this way:)

Official meeting minutes and especially records of decisions made <em class=rfc2119>must</em> be archived by W3C for future reference, and other persistent text-based discussions sponsored by the group pertaining to their work and intended to be referenceable by all group members <em class=rfc2119>should</em> be.
 			.

Reply:
Hmm, I'm not sure I like the missing predicate on the second clause better than the parenthetical. Although I'm totally fine with putting the MUST clause before the SHOULD one.

Reply:
Breaking the tie: I took in the proposed changed.